user/container/entry.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo -e
+
+echo "< Starting Kubestellar container >-------------------------"
```

Comment:
@ronenkat can you clarify. or provide a specific change. I'm not sure what to do here.

Reply:
Normally the YAMLs supplied with a Kubernetes workload include RBAC objects that enable the workload to work, and installing such a workload requires admin privileges because this installation operation includes adding those RBAC objects.